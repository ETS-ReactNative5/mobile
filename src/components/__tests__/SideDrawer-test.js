import 'react-native'
import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow';
import SideDrawer from '../SideDrawer'

it('renders correctly', () => {
  const navigationState = {
    children: [{ }],
    open: true
  }

  const renderer = new ShallowRenderer();

  const tree = renderer.render(
    <SideDrawer navigationState={navigationState} onNavigate={jest.fn}/>
  )
  expect(tree).toMatchSnapshot()
})
