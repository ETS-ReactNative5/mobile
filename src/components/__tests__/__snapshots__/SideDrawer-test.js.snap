// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Drawer
  closedDrawerOffset={-3}
  content={<UNDEFINED />}
  onClose={[Function]}
  onOpen={[Function]}
  open={true}
  openDrawerOffset={0.3}
  panCloseMask={0.2}
  side="right"
  styles={
    Object {
      "drawer": Object {
        "backgroundColor": "white",
        "elevation": 3,
        "shadowColor": "black",
        "shadowOpacity": 0.8,
        "shadowRadius": 3,
      },
    }
  }
  tapToClose={true}
  tweenHandler={[Function]}
  type="overlay"
>
  <DefaultRenderer
    navigationState={Object {}}
    onNavigate={[Function]}
  />
</Drawer>
`;
